src/components/Profile/ProfileFooter.tsx

```diff
@@ -1,45 +1,53 @@
 import classNames from 'classnames'
 import React, { useEffect, useState } from 'react'
 import { ChevronDown } from 'react-feather'
+import { Box, Flex, Text } from 'rebass'
 
 import colors from 'utils/colors'
 
+const WigglyChevron = ({ isAnimating }: { isAnimating: boolean }) => (
+  <Flex
+    className={classNames('justify-center mt2', {
+      'hvr-wobble-vertical hvr-wobble-vertical-animating': isAnimating,
+    })}
+  >
+    <ChevronDown className="mw-5" color={colors.primary} size="30px" />
+  </Flex>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)